streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorContextImpl.java

```diff
@@ -86,15 +85,15 @@ public StateStore getStateStore(final String name) {
         final StateStore global = stateManager.getGlobalStore(name);
         if (global != null) {
             if (global instanceof TimestampedKeyValueStore) {
-                return new TimestampedKeyValueStoreReadOnlyDecorator((TimestampedKeyValueStore) global);
+                return new TimestampedKeyValueStoreReadOnlyDecorator<>((TimestampedKeyValueStore<?, ?>) global);
```

Comment:
What's the purpose of adding generic here?

Reply:
To avoid warnings about "raw type" usage. Gradle does not fail on this, but IntelliJ highlights it, and it's best practice to avoid "raw types"